src/GovUk.Education.ExploreEducationStatistics.Data.Api/Services/PermalinkService.cs

```diff
@@ -81,10 +81,8 @@ public class PermalinkService : IPermalinkService
                 .OnSuccessCombineWith(_ => DownloadPermalink(permalinkId, cancellationToken))
                 .OnSuccess(async tuple =>
                 {
-                    var (permalink, universalTable) = tuple;
-
-                    PermalinkViewModel viewModel = await BuildViewModel(permalink, universalTable);
-                    return viewModel;
+                    var (permalink, table) = tuple;
```

Comment:
Wish you could destructure tuples in the lambda expression itself!

Reply:
Yeah that is a much needed addition! 🙂